ansible/roles/raspbian/bitcoin/templates/bitcoin.conf.j2

```diff
@@ -7,7 +7,8 @@
 # bitcoind options
 server=1
 daemon=1
-txindex=1
+#txindex=1
+prune=550
```

Comment:
This should be left as is. Obviously, this is a highly opinionated configuration and users of this role will need to configure what they want.